woocommerce/woocommerce-ios
# Order Details: Customer Info

Task #11 

To Test:
1. Go to Orders > Choose an order (Order #949 is a good example)
2. The "Customer Information" section should appear, with a title, Shipping details cell, and "Show billing" footer.
3. Tap anywhere on the "Show billing" footer to see it expand and show the Billing details cell.

Known Issues:
- some of the other section titles are stubbed in, which will make the table spacing look odd
- the "more" button and "email" button actions are not part of the code yet because I don't know if I've implemented the existing button actions as closures correctly or not. (See ShowHideBillingFooter code)
